lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -320,6 +320,41 @@ def self.update_release(api_token, owner_name, app_name, release_id, release_not
         end
       end
 
+      # updates release metadata
+      def self.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature)
+
+        release_metadata = nil
+        if dsa_signature.to_s != ''
+          release_metadata = {
+            dsa_signature: dsa_signature
+          }
+        end
+
+        if release_metadata.nil?
+          return
+        end
```

Comment:
This feels like a lot of lines for such a small logic. For now we don't have other metadata so there's no need to make the code complex. Why not just do this:
```ruby
return if dsa_signature.to_s == ''

release_metadata = { ... }
```

Reply:
Yeah for now, this release_metadata for only dsa_signature..
But yeah let's handle in future, if any new fields comes up.